target/linux/ath79/dts/qca9563_tplink_archer-a6-v2.dts

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9563_tplink_archer-x6-v2.dtsi"
+
+/ {
+	compatible = "tplink,archer-a6-v2", "qca,qca9563";
```

Comment:
If this is more or less the same file as target/linux/ath79/dts/qca9563_tplink_archer-c6-v2.dts you can also do this:

```
// SPDX-License-Identifier: GPL-2.0-or-later OR MIT

#include "qca9563_tplink_archer-c6-v2.dts"

/ {
	compatible = "tplink,archer-a6-v2", "qca,qca9563";
	model = "TP-Link Archer A6 v2 (EU/RU)";
};
```

If there are more changes needed you can add overwrite the settings.

Reply:
@hauke I don't really like deriving DTSes from other DTSes. It's always somewhat counter-intuitive; apart from that, somebody changing c6-v2 DTS will not be aware that he is also indirectly changing another device.

I'd be pleased if we would keep the policy of not doing this at all.